app/src/main/java/org/covidwatch/android/exposurenotification/FakeExposureNotification.kt

```diff
@@ -76,4 +76,6 @@ object RandomEnObjects {
             .setTotalRiskScore(Random.nextInt(8))
             .setTransmissionRiskLevel(Random.nextInt(8))
             .build()
+
+    var retrieved = false
```

Comment:
I believe, there is a way of avoiding this. That's a bad code practice so I'd prefer to solve the problem the other way

Reply:
What is the other way?